src/algorithms/eigsolvers/Arnoldi.m

```diff
@@ -191,6 +225,28 @@
                 V(:, alg.deflatedim + 1:end) = 0 * V(:, alg.deflatedim + 1:end);
                 ctr_inner = alg.deflatedim;
             end
+            
+            % process results
+            if nargout <= 1
+                varargout = {D};
+            else
+                if ~isnumeric(v0)
+                    for i = howmany:-1:1
+                        varargout{1}(:, i) = devectorize(V(:, i), v0);
+                    end
+                end
+                varargout{2} = D;
+                if nargout == 3
```

Comment:
The default `eigs` of Matlab only prints warnings if `flag` is not returned. Maybe we can do the same here, such that we can later more dynamically suppress the large amount of warnings?

Reply:
Done.